nexus-repository-r-it/src/test/java/org/sonatype/nexus/plugins/r/internal/RHostedIT.java

```diff
@@ -170,7 +186,7 @@ private void uploadPackages(String... names) throws IOException {
     assertThat(getAllComponents(repository), hasSize(names.length));
   }
 
-  private void uploadSinglePackage(String name) throws IOException {
-    client.put(format("%s/%s", PKG_GZ_PATH, name), fileToHttpEntity(name));
+  private HttpResponse uploadSinglePackage(String name) throws IOException {
+    return client.put(format("%s/%s", PKG_GZ_PATH, name), fileToHttpEntity(name));
```

Comment:
Question: does this consume the http response later? for example see https://github.com/sonatype/nexus-internal/blob/128cadc85f603d269f2c0b6d200028117ba00bce/private/testsuite/nexus-repository-testsupport-internal/src/main/java/org/sonatype/nexus/testsuite/testsupport/yum/YumClient.java#L100

Reply:
No, updated in 661a917b12dfb9f7554aeccbc23e3b295e57551f